couch/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
We do not. Reference this conversation for a more detailed discussion of why: https://github.com/DataDog/integrations-core/pull/38#discussion_r93923130